src/homepage.js

```diff
@@ -0,0 +1,119 @@
+const DATABASE = require("./database.json");
+
+const LOGO_PATH =
+  "https://raw.githubusercontent.com/denoland/deno/master/website/deno_logo.png";
+
+const homepageHTML = `
```

Comment:
If you mark this as HTML, Prettier will format it as such:


```suggestion
const homepageHTML = /* HTML */ `
```

Reply:
Will open a PR.